test/api/projects_api_test.rb

```diff
@@ -11,11 +11,33 @@ def app
   end
 
   def test_can_get_projects
-    user = FactoryBot.create(:user, :student, enrol_in: 0)
-    get with_auth_token('/api/projects', user)
+    # Create new student user
+    user = FactoryBot.create(:user, :admin)
+
+    # Create a dummy project
+    new_project = FactoryBot.create(:project)
+
+    # Perform Get
+    get with_auth_token "/api/projects/#{new_project.id}", new_project.user
```

Comment:
`new_project.user` you used here is not the admin `user` you created in line 14 but the default student user of the project you created therefore line 15 is not used. What this also means is that student user of a project has the authorisation to get the project's details. What I suggest is that you create a new `unit` by FactoryGirl, get a `project` from that unit by `project  = unit.active_projects.first` then use assign the student of that project `student = project.student' to perform the get.